test/v2

```diff
@@ -116,7 +116,6 @@ test_installDefaultJava() {
 
   assertEquals "${BUILD_DIR}/.jdk/bin/java" "$(command -v java)"
   assertTrue "A profile.d file should have been created." "[ -f ${BUILD_DIR}/.profile.d/jvmcommon.sh ]"
-  assertTrue "A pgconfig.jar should exist in the JDK" "[ -f ${BUILD_DIR}/.jdk/jre/lib/ext/pgconfig.jar ]"
```

Comment:
This assertion passed when OpenJDK 8 was always the default and contained `pgconfig.jar`. For OpenJDK > 8, that file doesn't exist. This caused this assertion to fail on `heroku-24`.

The presence of `pgconfig.jar` on OpenJDK 8 is verified in other tests already, so removal here is the simplest fix without a downside.